### PR TITLE
[ssd_generic] Get health status from Remaining_Life_Left field for virtium SSD

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
#### Description
For virtium SSD, sometimes there is no NAND_Endurance and Average_Erase_Count field in SSD smart attributes. We should use Remaining_Life_Left in such case.

#### Motivation and Context

To support SSD health on more virtium SSD

#### How Has This Been Tested?

Manual test
Unit test

### CODE REVIEW

**Comment by prgeor** [comment]
@Junchao-Mellanox can you paste the output of the CLI?

File: sonic_platform_base/sonic_ssd/ssd_generic.py [hunk 1]
```diff
@@ -137,11 +137,17 @@ def parse_virtium_info(self):
...
-                pass
-
+            if nand_endurance != NOT_AVAILABLE and avg_erase_count != NOT_AVAILABLE:
+                try:
+                    self.health = 100 - (float(avg_erase_count) * 100 / float(nand_endurance))
+                except (ValueError, ZeroDivisionError):
+                    pass
+            else:
+                try:
+                    self.health = float(self._parse_re('Remaining_Life_Left\s*\d*\s*(\d+?)\s+', self.vendor_ssd_info))
```
Comment (prgeor) [inline sonic_platform_base/sonic_ssd/ssd_generic.py:147]: @Junchao-Mellanox is the remaining life left in percentage?

Comment (Junchao-Mellanox) [reply sonic_platform_base/sonic_ssd/ssd_generic.py:147]: yes

**Comment by Junchao-Mellanox** [comment]
> @Junchao-Mellanox can you paste the output of the CLI?

There is no output change:

```
Device Model : StorFly VSF302XC016G-MLX1
Health       : 87.9%
Temperature  : 26C
```

**Review by liat-grozovik — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@prgeor kindly reminder to review following your comments

**Comment by Junchao-Mellanox** [comment]
@prgeor kindly reminder

**Review by prgeor — APPROVED** [review]

**Pull request merged**